CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Major: Added customizable shortcuts. (#2340)
+- Minor: Refactor variable names to reduce build warnings (#3369)
```

Comment:
This does not require a changelog entry at all, please remove it. Regular user doesn't ever build the app, so this is irrelevant.